### PR TITLE
Update Hyundai firmware in values.py for 2021 Sonata

### PR DESCRIPTION
**Pull request by erdardet**
Added firmware versions from a 2021 Hyundai Sonata bought in Southern California

dongle_ID  57205cbb3cb26844

A few routes are uploaded to my.comma.ai

### CODE REVIEW

**Comment by erdardet** [comment]
My note in Discord:
https://discord.com/channels/469524606043160576/524611978208215070/785806574987509790

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ class CAR:
   KONA = "HYUNDAI KONA 2020"
   KONA_EV = "HYUNDAI KONA ELECTRIC 2019"
   SANTA_FE = "HYUNDAI SANTA FE LIMITED 2019"
-  SONATA = "HYUNDAI SONATA 2020"
+  SONATA = "HYUNDAI SONATA 2020-2021"
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:31]: We don't update the years in the version string. Feel free to update the year in the README.md though!

Comment (erdardet) [reply selfdrive/car/hyundai/values.py:31]: Sounds good! Should I edit this PR / resubmit another PR? Or will these lines get modified before merge?

Comment (pd0wm) [reply selfdrive/car/hyundai/values.py:31]: You can just edit this PR.

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -164,16 +164,19 @@ class Buttons:
       b'\xf1\x00DN8_ SCC FHCUP      1.00 1.01 99110-L1000         ',
       b'\xf1\x00DN8_ SCC FHCUP      1.00 1.00 99110-L0000         ',
       b'\xf1\x00DN8_ SCC F-CU-      1.00 1.00 99110-L0000         ',
+      b'\xf1\x00DN8_ SCC F-CUP      1.00 1.00 99110-L0000         ',
     ],
     (Ecu.esp, 0x7d1, None): [
       b'\xf1\x00DN ESC \x01 102\x19\x04\x13 58910-L1300\xf1\xa01.02',
       b'\xf1\x00DN ESC \x06 104\x19\x08\x01 58910-L0100',
       b'\xf1\x8758910-L0100\xf1\x00DN ESC \x06 104\x19\x08\x01 58910-L0100\xf1\xa01.04',
+      b'\xf1\x8758910-L0100\xf1\x00DN ESC \a 104\x19\b\x01 58910-L0100\xf1\xa01.04',
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:173]:
```suggestion
      b'\xf1\x8758910-L0100\xf1\x00DN ESC \x07 104\x19\x08\x01 58910-L0100\xf1\xa01.04',
```

Comment (pd0wm) [reply selfdrive/car/hyundai/values.py:173]: `\a` == `\x07`, and `\b` is `\x08`. Let's use the full byte for consistency with other FW versions.

File: selfdrive/car/hyundai/values.py (hunk 1 above)
Comment (erdardet) [reply selfdrive/car/hyundai/values.py]: Edited this PR to keep line 31 unchanged from commaai source

**Comment by erdardet** [comment]
@pd0wm accepted your review and reverted the change in line 31 to keep commaai source

**Pull request merged**